.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "dotnet-format": {
+      "version": "4.1.131201",
```

Comment:
Let's use the latest release (5.1.225507), unless there is a good reason not to?

Reply:
fixed 👍